bin/capsule.js

```diff
@@ -10,41 +10,160 @@ const chalk = require('chalk');
 const aws = require('aws-sdk');
 const path = require('path')
 const Spinner = require('cli-spinner').Spinner;
+const { prompt } = require('inquirer');
+
 let cf;
 let s3;
 let cfr;
 
 //#############################################################################
 
+/*
+ * Globals
+ */
+const {
+  // AWS Access Key
+  AWS_ACCESS_KEY_ID,
+  // AWS secret key
+  AWS_SECRET_ACCESS_KEY,
+  // AWS profile name
+  AWS_PROFILE,
+  // AWS region
+  AWS_REGION
+} = process.env;
+
+/*
+ * References
+ * https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-describing-stacks.html#w2ab2c15c15c17c11
+ *
+ */
+const stack_states = [
+  'CREATE_COMPLETE',
+  'CREATE_FAILED',
+  'DELETE_COMPLETE',
+  'DELETE_FAILED',
+  'UPDATE_COMPLETE',
+  'UPDATE_FAILED',
+  'ROLLBACK_COMPLETE',
+  'ROLLBACK_FAILED',
+  'UPDATE_ROLLBACK_COMPLETE',
+  'UPDATE_ROLLBACK_FAILED',
+  'REVIEW_IN_PROGRESS'
+];
+
+const error_states = [
+  'CREATE_FAILED',
+  'DELETE_FAILED',
+  'UPDATE_FAILED',
+  'ROLLBACK_FAILED',
+  'UPDATE_ROLLBACK_FAILED'
+];
+
+const paths = {
+  base: `${__dirname}/../`,
+  ci_s3: 'ci/s3_cloudformation.cf',
+  ci: 'ci/codebuild_capsule.cf',
+  ci_project: 'ci/codebuild_project.cf',
+  cf_templates: 'templates/child_templates/',
+  web_template: 'templates/template.yaml',
+  base_config: 'config/capsule_init_questions.json',
+  ci_config: 'config/capsule_init_ci_questions.json',
+  aws_url: 'https://s3.amazonaws.com/',
+  output_config: 'capsule.json'
+}
+
+const projectParameters = {
+  site_config_params: {},
+  site_config_file: {},
+  site_config: {}
+}
+
+
+let last_time = new Date(new Date() - 1000);
+
+
+/**
+ * Takes a JSON config file, opens its, reads the contents
+ * passes it, and returns a JS object.
+ *
+ * @method getJsonFile
+ *
+ * @param {String} path
+ *
+ * @return {Object} data
+ */
+const getJsonFile = (path) => {
+  return new Promise((resolve, reject) => {
+    fs.readFile(path, 'utf8', (err, data) => {
+      if (err) reject(err);
+      else resolve(JSON.parse(data));
+    })
+  });
+}
+
+/**
+ * Pass in a config file in JSON format
+ * process and return an object
+ *
+ * @method parseJsonConfig
+ *
+ * @param {String} site_config_file
+ *
+ * @return {Object}
+ */
+const parseJsonConfig = async (site_config_file) => getJsonFile(site_config_file);
+
+
+/**
+ *
+ * Write config file out by taking the
+ * JSOn object and stringifying it
+ *
+ * @method writeConfigFile
+ *
+ * @param {Object} answers
+ *
+ * @return {void}
+ */
+const writeConfigFile = (answers) => {
+  fs.writeFile(paths.output_config, JSON.stringify(answers), function(err) {
+    if(err) {
+      return console.log(err);
```

Comment:
or `console.error(err)`

Reply:
Good spot. Have updated this to use the logIfVerbose function as well.